spoon/form/form.php

```diff
@@ -1151,8 +1151,7 @@ public function validate()
 			else
 			{
 				// get the submitted token
-				$data = $this->getField('form_token')->getMethod(true);
```

Comment:
what does the getMethod function do?

Reply:
This is the `getMethod()`
```
	/**
	 * Retrieve the form method or the submitted data.
	 *
	 * @return	string
	 * @param	bool[optional] $array	Should we return the submitted data?
	 */
	public function getMethod($array = false)
	{
		// we want to get the actual $_GET or $_POST data
		if($array)
		{
			// $_POST array
			if($this->method == 'post') return $_POST;

			// $_GET array
			return SpoonFilter::arrayMapRecursive('urldecode', $_GET);
		}

		// submitted via met get or post
		return $this->method;
	}
```
We submit `true` in the method, which means we want to fetch all the variables from $_POST or $_GET.
BUT: we call this method at start and it seems the $_POST and $_GET are always returning `[]` at start, so indeed there is no "form_token" to find.

**SOLUTIONS**:
1. We use `$this->getField('form_token')->getValue()`, which always returns the `form_token`.
2. Or we must look to set $_POST and $_GET at the start (before the form is submitted). Or look why this worked in the past and not anymore.